app/serializers/serialize_exercise.rb

```diff
@@ -13,7 +13,7 @@ def call
       type: exercise.tutorial? ? "tutorial" : exercise.git_type,
       title: exercise.title,
       icon_url: exercise.icon_url,
-      difficulty: "easy", # TOOD
+      difficulty: exercise.difficulty_category,
```

Comment:
Should the property be `difficulty_category`?

Reply:
Yeah - good shout. It'll also need updating at the JS side (camelcased) too. So either we do that now, or we add a TODO for it for later?